doc/docs/.vuepress/public/specs/operateurs-api-v3.yaml

```
@@ -448,6 +466,40 @@ components:
          - durée transmise supérieure à 7 fois la durée estimée. Exemple : 1h estimé vs 7h transmis
          La durée transmise est très largement supérieure par rapport à l'estimation. Le seuil est élevé pour prendre en compte d'éventuels embouteillage
 paths:
+  /auth/token:
+    post:
+      tags:
+      - auth
```

yamllint

[warning] wrong indentation: expected 8 but found 6

(indentation)